eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Content.php

```diff
@@ -57,11 +61,20 @@ public function mapConfig(array &$scopeSettings, $currentScope, ContextualizerIn
             $contextualizer->setContextualParameter('content.default_ttl', $currentScope, $scopeSettings['content']['default_ttl']);
 
             if (isset($scopeSettings['content']['tree_root'])) {
-                $contextualizer->setContextualParameter(
-                    'content.tree_root.location_id',
-                    $currentScope,
-                    $scopeSettings['content']['tree_root']['location_id']
-                );
+                if (isset($scopeSettings['content']['tree_root']['location_id'])) {
+                    $contextualizer->setContextualParameter(
+                        'content.tree_root.location_id',
+                        $currentScope,
+                        $scopeSettings['content']['tree_root']['location_id']
+                    );
+                }
+                if (isset($scopeSettings['content']['tree_root']['location_remote_id'])) {
```

Comment:
should we skip setting this is location_id is also set ?

Reply:
yeah, why not.